freqtrade/plugins/pairlist/rangestabilityfilter.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, exchange, pairlistmanager,
 
         self._days = pairlistconfig.get('lookback_days', 10)
         self._min_rate_of_change = pairlistconfig.get('min_rate_of_change', 0.01)
+        self._max_rate_of_change = pairlistconfig.get('max_rate_of_change', 0.99)
```

Comment:
i think the default for this should be None (obviously will need checking below before using this value).

`max_rate_of_change` is new - and 0.99 is pretty tight considering how crypto-markets work...

n the top 10 USDT volume pairs on binance, i get AXS/USDT with 2.7, ALICE/USDT with 3.0 and C98/USDT with a value of 38 at the moment (ok, that last one has been listed 5 days ago or so, but the others ...).


while there can be reasons to exclude these - it should be intentional and wouldbe surprising (and undesired -> a somehow breaking change) to have them excluded automatically by an update - as then it'd not be intentional.

Reply:
Agreed, and thanks for the suggestion. My intention was to set it so high that none would be excluded by default. So the 0.99 should've been something more like 99.0 but a value of `None` should suffice. Logically, I'm not sure if either 99.0 or None is a better option than the other. If one is better than the other, do enlighten me :)

Reply:
using None explicitly unsets it.

we won't know what markets people will use freqtrade in (and also not how markets will move in the future).
with a sufficient lookback on cheap, or newly listed pairs - you can easily get values of > 99 (i got a value of 38 yesterday).
As such - i'd explicitly unset this - and only if users explicitly set the value - it'll apply. Otherwise we'll introduce magic constants which we are not sure will hold in the future.

Reply:
> using None explicitly unsets it.
...
> As such - i'd explicitly unset this - and only if users explicitly set the value - it'll apply. Otherwise we'll introduce magic constants which we are not sure will hold in the future.

I understand... I appreciate you "holding my hand" through some of these things. The strategy I'm working on is dependent on this enhancement (have been testing with my original quick and dirty filter), so when that cash starts rolling in I'll be sure to hit that support button :)